cmd/cri-containerd/options/options.go

```diff
@@ -105,63 +100,52 @@ func NewCRIContainerdOptions() *CRIContainerdOptions {
 
 // AddFlags adds cri-containerd command line options to pflag.
 func (c *CRIContainerdOptions) AddFlags(fs *pflag.FlagSet) {
+	defaults := defaultConfig()
 	fs.StringVar(&c.ConfigFilePath, configFilePathArgName,
 		"/etc/cri-containerd/config.toml", "Path to the config file.")
```

Comment:
defaults.ConfigFilePath?

Reply:
`ConfigFilePath` is not part of config and default config. But I can make this a const. :)